src/main.rs

```diff
@@ -17,22 +18,25 @@ const VERSION: &str = concat!(
 #[derive(Debug, Parser)]
 #[clap(name = "spin cloud-gpu", version = VERSION)]
 pub enum App {
-    /// Deploy the fermyon-cloud-gpu Spin app to act as a cloud GPU proxy.
+    /// Deploy the Fermyon Cloud GPU Spin App to act as a cloud GPU proxy.
     Init,
-    /// Destroy the fermyon-cloud-gpu Spin app.
+    /// Rotate the Auth Token for your existing Fermyon Cloud GPU
+    RotateAuthToken,
```

Comment:
```suggestion
    RotateToken,
```

A slightly shorter version to save some keystrokes.